examples/import-go/age-calculator_test.go

```diff
@@ -0,0 +1,25 @@
+package main
```

Comment:
I would rename to `age_calculator_test.go`

Reply:
ok

Reply:
went for package name `age_calculator` because I still want to be able to call `main`! good idea to use a separate package per example.. I did that to all of them

Reply:
wanna rename others too? (without hyphens)